vpr/src/draw/breakpoint.h

```diff
@@ -0,0 +1,81 @@
+#ifndef BREAKPOINT_H
+#define BREAKPOINT_H
+
+#include <vector>
+#include <string>
+
+#include "move_transactions.h"
+#include "expr_eval.h"
+
+class breakpoint {
```

Comment:
The convention we follow is that classes should be capitalized (i.e. `Breakpoint`).

Reply:
resolved